recipes/brainreg-segment/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "brainreg-segment" %}
+{% set version = "0.2.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/brainreg-segment-{{ version }}.tar.gz
+  sha256: c3b479a5d18b4af156c6b487f5e87e3c700adfd31d3a090067f62392f4a98d3a
+
+build:
+  entry_points:
+    - brainreg-segment = brainreg_segment.segment:main
```

Comment:
The command is not tested because it is actually launching napari with the plugin docked, so the CI hangs forever.

Reply:
That’s ok. It’s essentially a deprecated way of calling the tool. It existed before napari had plugins.